crypto/keys/der/der_test.go

```diff
@@ -61,22 +63,27 @@ func TestFromProto(t *testing.T) {
 			wantErr:  true,
 		},
 	} {
-		signer, err := FromProto(test.keyProto)
-		if gotErr := err != nil; gotErr != test.wantErr {
-			t.Errorf("%v: FromProto(%#v) = (_, %q), want (_, nil)", test.desc, test.keyProto, err)
-			continue
-		} else if gotErr {
-			continue
-		}
-
-		// Check that the returned signer can produce signatures successfully.
-		if err := testonly.SignAndVerify(signer, signer.Public()); err != nil {
-			t.Errorf("%v: SignAndVerify() = %q, want nil", test.desc, err)
-		}
+		t.Run(test.desc, func(t *testing.T) {
+			t.Parallel()
+
+			signer, err := FromProto(test.keyProto)
```

Comment:
I think this will fall foul of https://github.com/golang/go/wiki/CommonMistakes#using-goroutines-on-loop-iterator-variables.

Reply:
good spot, ta! (x_N_)